python/python/lance/dataset.py

```diff
@@ -827,6 +827,7 @@ def create_index(
             "dot",
         ]:
             raise ValueError(f"Metric {metric} not supported.")
+        kwargs["metric"] = metric.lower()
```

Comment:
Why is this needed if we pass `metric` to `create_index` down below?

Reply:
Good catch. This was a left-over from debugging. I will file another PR to fix this one.